ui/raidboss/data/02-arr/raid/t1.ts

```diff
@@ -1,9 +1,16 @@
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
 import Conditions from '../../../../../resources/conditions';
 import NetRegexes from '../../../../../resources/netregexes';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 
-export default {
+interface Data extends RaidbossData {
```

Comment:
It would be better to expect built-in raid data type so user using webpack to build their raidboss.js could import the

Reply:
I suspect that you are the only person writing personal triggers and using webpack.  :P

Just to clarify, when you said "expect built-in data type", did you mean "export"?  Are you suggesting to `export interface Data`?  If so, I don't see any reason not to.

Reply:
yes, that's a typo, I mean `interface Data extends RaidbossData`

In fact, I copy most of type definition from @MaikoTan